demisto_sdk/commands/format/update_generic.py

```diff
@@ -463,3 +463,43 @@ def check_server_version(self):
                 else:
                     self.data[self.from_version_key] = current_from_server_version
                     self.data.pop(self.json_from_server_version_key)
+
+    def adds_period_to_description(self):
+        """Adds a period to the end of the descriptions
+        if it does not already end with a period."""
+        logger.info("Adds a period to the end of the description")
+
+        def _is_url(value: str):
+            str_to_check = value.split()[-1]
```

Comment:
what if `value` fails?

Reply:
did you mean what if `value` equal to empty string?
it's going to fail, 
because of this i added in the `_add_period` function `if value` before i call to `_is_url` function

Reply:
cool, this is worth a short comment - `only called after checking value isn't empty`

Reply:
Done